packages/aws-serverless/rollup.npm.config.mjs

```diff
@@ -9,6 +9,10 @@ export default [
       entrypoints: ['src/index.ts', 'src/awslambda-auto.ts'],
       // packages with bundles have a different build directory structure
       hasBundles: true,
+      packageSpecificConfig: {
+        // Used for our custom eventContextExtractor
+        external: ['@opentelemetry/api'],
```

Comment:
just out of curiosity, why is this needed?

Reply:
Package build structure isn't right without this. It pulls this in under its own node_modules folder.

Reply:
👍